apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -407,63 +409,90 @@ describe('progressReduxTest', () => {
+      });
 
-    it('returns LevelStatus.perfect for completed level', () => {
-      const level = {
-        ids: [123]
-      };
-      const levelProgress = {
-        123: TestResults.ALL_PASS
-      };
-      const status = statusForLevel(level, levelProgress);
-      assert.strictEqual(status, LevelStatus.perfect);
-    });
+      it('returns LevelStatus.perfect for completed level', () => {
+        const level = {
+          ids: [123]
+        };
+        const levelProgress = {
+          123: TestResults.ALL_PASS
+        };
+        const status = statusForLevel(level, levelProgress);
+        assert.strictEqual(status, LevelStatus.perfect);
+      });
 
-    it('returns LevelStatus.not_tried for level with no progress', () => {
-      const level = {
-        ids: [123]
-      };
-      const levelProgress = {
-        999: TestResults.ALL_PASS
-      };
-      const status = statusForLevel(level, levelProgress);
-      assert.strictEqual(status, LevelStatus.not_tried);
-    });
+      it('returns LevelStatus.not_tried for level with no progress', () => {
+        const level = {
+          ids: [123]
+        };
+        const levelProgress = {
+          999: TestResults.ALL_PASS
+        };
+        const status = statusForLevel(level, levelProgress);
+        assert.strictEqual(status, LevelStatus.not_tried);
+      });
 
-    it('returns LevelStatus.locked for a locked peer_review stage', () => {
-      const level = {
-        kind: LevelKind.peer_review,
-        locked: true
-      };
-      const levelProgress = {};
-      const status = statusForLevel(level, levelProgress);
-      assert.strictEqual(status, LevelStatus.locked);
-    });
+      it('returns LevelStatus.locked for a locked peer_review stage', () => {
+        const level = {
+          kind: LevelKind.peer_review,
+          locked: true
+        };
+        const levelProgress = {};
+        const status = statusForLevel(level, levelProgress);
+        assert.strictEqual(status, LevelStatus.locked);
+      });
 
-    it('returns LevelStatus.perfect for a completed peer_review stage', () => {
-      const level = {
-        kind: LevelKind.peer_review,
-        locked: false,
-        status: LevelStatus.perfect
-      };
-      const levelProgress = {};
-      const status = statusForLevel(level, levelProgress);
-      assert.strictEqual(status, LevelStatus.perfect);
+      it('returns LevelStatus.perfect for a completed peer_review stage', () => {
+        const level = {
+          kind: LevelKind.peer_review,
+          locked: false,
+          status: LevelStatus.perfect
+        };
+        const levelProgress = {};
+        const status = statusForLevel(level, levelProgress);
+        assert.strictEqual(status, LevelStatus.perfect);
+      });
+      it('returns LevelStatus.completed_assessment for assessment level when experiment is on', () => {
+        sinon.stub(experiments, 'isEnabled').returns(true);
```

Comment:
handy tip: if you need to repeat steps (like stubbing and restoring a method here) for multiple tests, you can create a `describe` group:

```
describe('Level.completed_assessment', () => {
  beforeEach(() => {
    sinon.stub(experiments, 'isEnabled').returns(true);
  });

  afterEach(() => {
    experiments.isEnabled.restore();
  });

  it('does something', () => {
  });
});
```

you don't need to make this change here since there are only 2 tests stubbing `experiments.isEnabled()`, but wanted to pass this along!